package.json

```diff
@@ -103,7 +103,8 @@
     "typescript": "^5.5.3"
   },
   "overrides": {
-    "cookie": "^0.7.0"
+    "cookie": "^0.7.0",
+    "npm": "^10.9.0"
```

Comment:
can the error of the test come from this change?
https://github.com/affinidi/affinidi-cli/actions/runs/13234625987/job/36937249006?pr=453

Reply:
link to the [PR](https://github.com/affinidi/affinidi-cli/pull/455) that should be merged prior this one, to unblock the pipeline